README.md

```diff
@@ -14,6 +14,7 @@ GEI-CLI is continuing to expand what it can support. However, it supports the fo
 
 * Azure DevOps -> GitHub Enterprise Cloud migrations
 * GitHub Enterprise Cloud -> GitHub Enterprise Cloud migrations
+* GitHub Enterprise Server -> GitHub Enterprise Cloud migrations
```

Comment:
Is it too soon to call this GHES->GHEC a "supported" path?

Reply:
might be worth adding (>3.4)

to ensure it is noted for customers.